airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py

```diff
@@ -157,8 +157,8 @@ def parse_records(
                         record_field: self._to_output_value(avro_format, schema_field_name_to_type[record_field], record[record_field])
                         for record_field, record_value in schema_field_name_to_type.items()
                     }
-        except Exception as exc:
-            raise RecordParseError(FileBasedSourceError.ERROR_PARSING_RECORD, filename=file.uri, lineno=line_no) from exc
+            except Exception as exc:
+                raise RecordParseError(FileBasedSourceError.ERROR_PARSING_RECORD, filename=file.uri, lineno=line_no) from exc
```

Comment:
can you add a test case to [avro_scenarios](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/unit_tests/sources/file_based/scenarios/avro_scenarios.py) to codify the new end-to-end behavior?

The new behavior isn't obvious just from the PR and the unit tests. will this cause the sync to stop early or will this only change the error message? A new scenario will help document and ensure we don't introduce regressions in the future